test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -670,19 +670,21 @@ def init_servers(cls, init_server_func=None, skip=None):
         if init_server_func is None:
             init_server_func = cls.init_server
         if 'servers' in cls.conf:
-            if 'comms' not in cls.conf:
-                cls.conf['comms'] = cls.conf['servers']
-            if 'schedulers' not in cls.conf:
-                cls.conf['schedulers'] = cls.conf['servers']
-            if 'moms' not in cls.conf:
-                cls.conf['moms'] = cls.conf['servers']
+            server_param = cls.conf['servers']
+            if 'comms' not in cls.conf and 'comm' not in cls.conf:
+                cls.conf['comms'] = server_param
+            if 'schedulers' not in cls.conf and 'scheduler' not in cls.conf:
+                cls.conf['schedulers'] = server_param
+            if 'moms' not in cls.conf and 'mom' not in cls.conf:
+                cls.conf['moms'] = server_param
         if 'server' in cls.conf:
-            if 'comm' not in cls.conf:
-                cls.conf['comm'] = cls.conf['server']
-            if 'scheduler' not in cls.conf:
-                cls.conf['scheduler'] = cls.conf['server']
-            if 'mom' not in cls.conf:
-                cls.conf['mom'] = cls.conf['server']
+            server_param = cls.conf['server']
+            if 'comms' not in cls.conf and 'comm' not in cls.conf:
```

Comment:
we should not check 'comms' here, if only server is given then we should set only 'comm' not servers.